backend/src/test/java/ai/verta/modeldb/CommitTest.java

```diff
@@ -714,6 +714,8 @@ public void getCommitComponentTest() {
     LOGGER.info("Get commit blob test end................................");
   }
 
+  private static long time = Calendar.getInstance().getTimeInMillis();
```

Comment:
please add this variable on top of the class

Reply:
fixed